plugins/woocommerce/client/admin/config/core.json

```diff
@@ -12,6 +12,7 @@
 		"shipping-setting-tour": true,
 		"homescreen": true,
 		"marketing": true,
+		"multichannel-marketing": false,
```

Comment:
Hey @ecgan Do you plan to turn on this in a follow-up PR? I think we need to enable it before 7.0 code freeze.

Reply:
Yes, we want it to be off by default for now. This UI revamp is just the first step. There shall be more iterations adding more features to the Marketing page, and we shall turn it on by default when those features are done. More info is available here: pe2C5g-bI-p2.